src/NHPersistenceCompatibilityTests/Common/NugetHelper.cs

```diff
@@ -2,25 +2,31 @@
 using System.Collections.Generic;
 using System.Linq;
 using System;
+using System.Configuration;
 
 namespace Common
 {
     public class NugetHelper
     {
-        string packageSource;
-        string fallbackPackageSource;
+        IEnumerable<string> packageSources;
 
-        public NugetHelper(string packageSource = "https://packages.nuget.org/api/v2", string fallbackPackageSource = "https://www.myget.org/F/particular/")
+        public NugetHelper()
         {
-            // Todo: Load this package source from the machine config
-            // Todo: Support multiple package sources (e.g. Build server with Myget and Nuget)
-            this.packageSource = packageSource;
-            this.fallbackPackageSource = fallbackPackageSource;
+            LoadNugetSourcesFromConfig();
+        }
+
+        private void LoadNugetSourcesFromConfig()
+        {
+            var machineSettings = Settings.LoadDefaultSettings(new PhysicalFileSystem("C:\\"), null, null);
```

Comment:
It's the machine-wide Nuget config settings. The same as if you ran Nuget from the CLI. A better name might be `cliSettings`?